apps/frontend/core-dapp/src/features/portfolio/PositionsAndHistoryNext.tsx

```diff
@@ -0,0 +1,100 @@
+import { t } from '@lingui/macro'
+import { Box, centered, media, spacingIncrement } from 'prepo-ui'
+import { useState } from 'react'
+import styled, { useTheme } from 'styled-components'
+import FilterButton from '../../components/Filter/FilterButton'
+import Tabs from '../../components/Tabs'
+import History from '../history/History'
+import Positions from '../position/PositionsNext'
+
+const StyledTabs = styled(Tabs)`
+  &&&& {
+    .ant-select-selection-item {
+      padding-right: 20px;
+    }
+
+    .ant-select-selection-item,
+    .ant-tabs-tab-btn {
+      font-size: ${({ theme }): string => theme.fontSize.lg};
+    }
+
+    .ant-select-arrow {
+      left: 100%;
+      padding: 0 ${spacingIncrement(6)};
+      transform: translate(-100%, -50%);
+    }
+
+    .ant-tabs-nav {
+      flex-direction: column;
+      margin-bottom: 0;
+      ${media.desktop`
+        flex-direction: row;
+      `}
+    }
+
+    .ant-tabs-extra-content {
+      ${media.desktop`
+        width: auto;
+        padding: 0 ${spacingIncrement(30)};
+      `}
+      align-items: flex-end;
+      display: flex;
+      justify-content: flex-end;
+      padding: ${spacingIncrement(20)} ${spacingIncrement(37)};
+      width: 100%;
+    }
+
+    .ant-tabs-tab {
+      display: flex;
+      flex: 1;
+      padding: ${spacingIncrement(8)} 0;
```

Comment:
Then remove padding here:

```suggestion
      padding: 0;
```